resources/dicts/patrols/plains/border/greenleaf.json

```diff
@@ -31,13 +36,4753 @@
+                        "mutual": false,
+                        "values": ["dislike"],
+                        "amount": 20
+                    }
+                ],
+                "injury": [
+                    {
+                        "cats": ["p_l"],
+                        "injuries": ["battle_injury"],
+                        "scars": ["SCRATCHSIDE"]
+                    },
+                    {
+                        "cats": ["p_l"],
+                        "injuries": ["torn ear"],
+                        "scars": ["RIGHTEAR"]
+                    }
+                ],
+                "history_text": {
+                    "scar": "m_c was scarred up by a fight {PRONOUN/m_c/subject} had with a flirtatious rogue.",
+                    "reg_death": "The wounds obtained fighting a flirtatious rogue eventually killed m_c.",
+                    "lead_death": "wounds obtained fighting a flirtatious rogue eventually killed m_c"
+                },
+                "outsider_rep": -1
+            }
+        ],
+        "antag_success_outcomes": [
+            {
+                "text": "Oh yeah, p_l's <i>real</i> cute, {PRONOUN/p_l/subject} {VERB/p_l/reply/replies}, stalking round the loner and picking up the prey lying very poorly concealed behind {PRONOUN/n_c:0/object}. And p_l's <i>up to</i> a border patrol. c_n thanks n_c:0 for {PRONOUN/n_c:0/poss} donation to the fresh-kill pile. Now get back on the right side of those border markers. n_c:0 doesn't think p_l's much fun.",
+                "exp": 15,
+                "weight": 5,
+                "art": "mtn_hunt_kea_leafbare_parentchildapprentice1",
+                "new_cat": [
+                    ["age:adult", "loner", "meeting"]
+                ],
+                "prey": ["medium"],
+                "relationships": [
+                    {
+                        "cats_from": ["n_c:0"],
+                        "cats_to": ["p_l"],
+                        "mutual": false,
+                        "values": ["jealous", "respect"],
+                        "amount": 10
+                    },
+                    {
+                        "cats_from": ["n_c:0"],
+                        "cats_to": ["p_l"],
+                        "mutual": false,
+                        "values": ["dislike"],
+                        "amount": 5
+                    },
+                    {
+                        "cats_from": ["p_l"],
+                        "cats_to": ["n_c:0"],
+                        "mutual": false,
+                        "values": ["platonic", "dislike"],
+                        "amount": 10
+                    }
+                ],
+                "outsider_rep": -2
+            },
+            {
+                "text": "n_c:0 is on c_n's territory. Get off it. Wooooow, n_c:0 replies, rude. Besides, they've only just met, it's a little quick for anyone to get off. s_c blushes with what is definitely <i>only</i> righteous fury. GET OUT!",
```

Comment:
might be liiitle too much innuendo here

Reply:
ngl, I love this line, can I post it in dev game talk and run it past the other devs? I know the patrol definitely is full of innuendo but I like the prissy personality it gives the n_c

Reply:
Yeah we can run it by other devs.  Imo "get off" being such a direct innuendo for sex is what I think makes it a tad questionable.

Reply:
Fixed!